### PR TITLE
Move auth into a separate package

### PR DESCRIPTION
**Pull request by serjonya-trili**
## Types of changes

- [ ] Bugfix
- [ ] New feature
- [x] Refactor
- [ ] Breaking change
- [ ] UI fix

### CODE REVIEW

File: packages/typescript-config/tsconfig.json [hunk 1]
```diff
@@ -10,11 +10,11 @@
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true,
     "module": "ESNext",
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
```
Comment (asiia-trilitech) [inline packages/typescript-config/tsconfig.json:13]: What is the reasoning behind modeResolution change?

Comment (asiia-trilitech) [reply packages/typescript-config/tsconfig.json:13]: And for the removing `incremental: true` 🙂

**Review by asiia-trilitech — APPROVED** [review]
LGTM - thank you for the change, Sergey!

File: .yarn/releases/yarn-4.3.1.cjs
[the remark this answers is not in the document]
Comment (serjonya-trili) [reply .yarn/releases/yarn-4.3.1.cjs:1]: yes

File: packages/typescript-config/tsconfig.json (hunk 1 above)
Comment (serjonya-trili) [reply packages/typescript-config/tsconfig.json:13]: TL; DR; it doesn't work otherwise

**Pull request merged**